src/UserDetails.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import axios from './plugins/axios';
+import User from './User';
+
+export default class UserDetails extends React.Component {
+  state = {
+    user: {},
+    userComments: []
+  };
+  componentDidMount() {
+    this.displayUserDetails();
+  }
+  getUserComments = async (id) => await axios.get(`/item/${id}.json`);
+
+  displayUserDetails = async () => {
+    const userId = this.props.match.params.id;
+    const response = await axios.get(`/user/${userId}.json`);
+    this.setState({ user: response.data });
+  };
+
+  render() {
+    const { user } = this.state;
+
+    return (
+      <div>
```

Comment:
@VivianDoreen It's a good idea to use fragments if the html element does not serve any purpose e.g styling